signer-service/src/api/services/transactions/stellar/offrampTransaction.ts

```diff
@@ -124,6 +121,27 @@ export async function buildPaymentAndMergeTx(
     createAccountTransaction: createAccountTransaction.toEnvelope().toXDR().toString('base64'),
     paymentTransaction: paymentTransaction.toEnvelope().toXDR().toString('base64'),
     mergeAccountTransaction: mergeAccountTransaction.toEnvelope().toXDR().toString('base64'),
-    fundingAccountSequence,
+    expectedSequenceNumber: String(expectedSequenceNumber),
   };
 }
+
+async function getFutureShiftedLedgerSequence(horizonServer: HorizonServer, shiftAmount = 32) {
```

Comment:
I think I wouldn't make the `shiftAmount` configurable. There is no scenario in which you want to use a number different than `32`.